point_of_sale/shop.rst

```diff
@@ -6,5 +6,4 @@ Shop Features
    :titlesonly:
 
    shop/invoice
-   shop/refund
```

Comment:
It looks to me like your moved this content in getting_started. If so -> redirect rule.

Reply:
You're right.